Automattic/Gridicons-iOS
# Add Ruby configuration

Adds the Ruby configuration used for other iOS projects.

**To Test**
After merging #46, run `bundle install && rake dependencies && rake gen`. It should install rake using `bundler` and put it in the `vendor` directory. From there, it will use the bundled `rake` to run the required commands.
